github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml

```diff
@@ -1592,3 +1592,28 @@ spec:
   endpoints:
   - port: web
     path: "/metrics"
+---
+# PodMonitor to scrape prometheus metrics from each ephemeral Prometheus instance of each prow-job kubevirtci cluster
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: prometheus-stack-federation
+  namespace: default
+  labels:
+    app: kube-prometheus-stack-prometheus-federation
+
+    release: "prometheus-stack"
```

Comment:
So far we have been using `group: kubevirtci` for the `serviceMonitorSelector.matchLabels` entry in the Prometheus instance config here https://github.com/kubevirt/project-infra/blob/master/github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml#L1274. We didn't use PodMonitors so far, so I think it would be good to keep `group: kubevirtci` and use it for `podMonitorSelector.matchLabels` instead of the default `release: "prometheus-stack"` here https://github.com/kubevirt/project-infra/blob/master/github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml#L1279 wdyt?

Reply:
Ok, I will used only `group: kubevirtci`